cypress/integration/loans/loans-claim-returned.spec.js

```diff
@@ -0,0 +1,177 @@
+import moment from 'moment';
+import uuid from 'uuid';
+import devTeams from '../../support/dictionary/devTeams';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import TestTypes from '../../support/dictionary/testTypes';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import AppPaths from '../../support/fragments/app-paths';
+import Loans from '../../support/fragments/loans/loansPage';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
```

Comment:
'InventoryInstance' is defined but never used

Reply:
Done.